### ISSUE TITLE
Trials HD broken on Canary builds after June 20th

### ISSUE THREAD

**Issue by legend800** [xenia-canary/xenia-canary#67]
Trials HD now crashes during launch 100% of the time. Before it would load fine 4/5 times. 

- Broken on a build with all these Master merges: https://github.com/xenia-canary/xenia-canary/commit/87fd772393abf077d2ddf2dc3a0daa34dd827c7b
- Timing related - doesn't crash on debug builds because they're slow and don't trigger issue
- Last working build Jun20th - https://github.com/xenia-canary/xenia-canary/releases?page=5

![unknown](https://user-images.githubusercontent.com/20983844/187098674-820aa188-47e5-4dbc-bb33-5bd73c6660de.png)
- Changes from master all look edram/memexport related. That looks like its supposed to be physical memory
- Discord discussion:
      - "the reason for Trials HD dying spectacularly seems to be that it ends up with a $ffffffff value somewhere and then just keeps increasing it, as it craps out trying to read/write above a 4GB offset somehow."
      - "I'm not sure about trials because I had crashes in like 20 different places. But that might be right. Some global variable increased by any thread"

Note: The game required Readback Resolve to fix brightness/bloom issues but at a cost of perf. With recent Canary perf improvements, the game should be fully playable now for the first time, if we can fix the crash regression.

**Comment by Margen67** [comment]
Does this still occur?

**Comment by legend800** [comment]
Yes, worse now. Splash screen and audio don't even show now. 5/5 black screens.
[HD.log](https://github.com/xenia-canary/xenia-canary/files/11302931/HD.log)

Fine no Master:
<img width="963" alt="2023-04-22 23_06_18-Xenia (master@8aaa6f1f7 on Apr 19 2023) _  5841095A v2 0 1  Trials HD _Direct3D " src="https://user-images.githubusercontent.com/20983844/233822950-7e91a60a-3acb-4107-ad9e-a77f6160f9cb.png">

Related, looks like Trial Evo is also regressed. Very unstable, even on Master. Before it was just missing gfx. Now you can barely to the menus. 6/6 crashes by menu. Can't reach gameplay. 

<img width="961" alt="2023-04-22 22_57_07-Xenia-canary (canary_experimental@fecd1e6a4 on Apr 16 2023) _  58410AAD v3 0 3  " src="https://user-images.githubusercontent.com/20983844/233822622-d00881d4-9852-4013-9612-dec0a944528f.png">

[evo.log](https://github.com/xenia-canary/xenia-canary/files/11302929/evo.log)

**Comment by Gliniak** [comment]
It's better on latest versions. Trials HD sometimes still crash on boot, but it can frequently reach stability. Evolution is a bit more fragile, but it still can reach menu. Closing as fixed.
